app/src/main/java/com/example/sharingang/utils/DateHelper.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.sharingang.utils
+
+import java.util.*
+
+object DateHelper {
+
+    private val monthsMap = hashMapOf(
+        0 to "Jan",
+        1 to "Feb",
+        2 to "Mar",
+        3 to "Apr",
+        4 to "May",
+        5 to "Jun",
+        6 to "Jul",
+        7 to "Aug",
+        8 to "Sep",
+        9 to "Oct",
+        10 to "Nov",
+        11 to "Dec"
+    )
+
+    /**
+     * formats a date for message timestamps
+     *
+     * @param date the date to format
+     * @return the formatted date
+     */
+    fun formatMessageDate(date: Date): String {
+        val currentDate = Date()
+        val today = Triple(currentDate.year, currentDate.month, currentDate.day)
+        val messageTime = Triple(date.year, date.month, date.day)
+        val monthStr = monthsMap[date.month]
+        return (
+            when {
+                today == messageTime -> "Today, ${date.hours}:${
```

Comment:
String resource? Would be easier for possible future translation

Reply:
If you know a way of accessing the string resource file without a context I would gladly change. However I don't know how to do that.

Reply:
[https://stackoverflow.com/questions/4253328/getstring-outside-of-a-context-or-activity](url)

Reply:
Thanks! The website says 404 not found but I will look it up 👌🏻